packer/plugin.go

```diff
@@ -219,14 +235,19 @@ func (c *PluginConfig) discoverSingle(glob string) (map[string]string, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	prefix := ""
 	res := make(map[string]string)
-
-	prefix := filepath.Base(glob)
-	prefix = prefix[:strings.Index(prefix, "*")]
+	// Sort the matches so we add the newer version of a plugin last
+	sort.Strings(matches)
```

Comment:
Is this always guaranteed to be true? I think we use go-version for checking version things, which has a sort method https://github.com/hashicorp/go-version#version-sorting

Reply:
disregard